### ISSUE TITLE
(FIXED)  ASN.1 not expanding correctly

### ISSUE THREAD

**Issue by sjeon87** [proj3rd/lib3rd#86]
Spec: 36.331
Message/IE:
- `BCCH-BCH-MessageType`
- `SystemInformationBlockType1-BR-r13`
- etc.

```
BCCH-BCH-MessageType               ::= MasterInformationBlock
SystemInformationBlockType1-BR-r13 ::= SystemInformationBlockType1
```

**Comment by sjeon87** [comment]
Seems it does not handle direct `typeAssignment` of `definedType` properly

![image](https://user-images.githubusercontent.com/6747391/78025206-8f9ce680-7394-11ea-8ed8-418b1a587a62.png)

**Comment by sjeon87** [comment]
Fixed at efe9717f8cef75daf6701e77a93769c6fdf14d06

- [ ] Add to test case

**Comment by sjeon87** [comment]
3f39c2947d40976cb9e062b310805efaee2e2d0d
